why.md

```diff
@@ -8,24 +8,110 @@ editor: markdown
 dateCreated: 2023-02-09T00:04:00.000Z
 ---
 
-# Why use a 40% keyboard?
 
-On a full-size keyboard, pressing less frequent keys involves moving your hands or stretching your fingers to hit small distant targets and then reorienting to the home row. That reaching is slow and can lead to hand issues; a full-size keyboard also uses more desk space and is hard to throw in a bag for tablet use in a cafe, classroom or plane.
+Although people have different reasons for using 40% keyboards, some common reasons are:
```

Comment:
This line seems obvious and unnecessary. If the title is "Why use a 40%?" you can just start listing reasons right away.

Reply:
Partly to say "Why you like them isn't wrong, but we'll just talk about these three. " And as a more gradual intro to the piece. 

That has been reworked many times. "While people..." was another option. I'll think on it more.